woocommerce/woocommerce-gutenberg-products-block
# Remove author, sticky, parents attributes from the Product Collection block in patterns

## What

In this PR the unnecessary attributes are removed from the Product Collection block used in patterns.

## Why

These attributes were removed from the block itself:
- `author` in https://github.com/woocommerce/woocommerce-blocks/pull/11427
- `sticky` and `parents` in https://github.com/woocommerce/woocommerce-blocks/pull/11259
and they were leftover in the patterns. This PR cleans that up.

## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Go to Editor
2. Add following patterns
  - Product Collection 3 Columns
  - Product Collection 4 Columns
  - Product Collection 5 Columns
  - Product Collection: Featured Products 5 Columns
  - Product Collection Full Grid
  - Product Collection Grid
  - Product Collection Rows
  - Product Collection Simple Grid
  - Product Gallery
3. Make sure they are rendered correctly and there's no errors/warnings
4. Save and go to frontend
5. Make sure they are rendered correctly and there's no errors/warnings


## Screenshots or screencast



| Before | After |
| ------ | ----- |
|        |       |

## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:

## Changelog


> Patterns: remove unused `author`, `sticky`, and `parents` attributes from the Product Collection block in patterns
